modules/internal/ChapelDistribution.chpl

```diff
@@ -1,15 +1,15 @@
 /*
  * Copyright 2004-2017 Cray Inc.
  * Other additional copyright holders may be indicated within.
- * 
+ *
```

Comment:
In the future, when doing a big trailing whitespace cleanup, I'd recommend doing it in a PR of its own, merging it as trivial, and then pulling it into your working branch rather than having it clutter up an interesting PR...